src/main/java/com/codeborne/selenide/webdriver/WebDriverBinaryManager.java

```diff
@@ -58,20 +62,35 @@ private void setupFirefox() {
 
   private void cacheMeIfYouCan(String systemPropertyName, Runnable webdriverSetup) {
     if (webdriverIsAlreadyInitialized(systemPropertyName)) {
+      log.info("Skip: webdriver is already initialized: " + System.getProperty(systemPropertyName));
```

Comment:
Do we really need it on `info` level?
I think debug will be enough

Reply:
I planned to delete this commit in future. It's temporary needed to debug the problem.